Manifests/ManagedPreferencesApplications/com.github.macadmins.Nudge.plist

```diff
@@ -13,9 +13,9 @@
 	<key>pfm_format_version</key>
 	<integer>1</integer>
 	<key>pfm_last_modified</key>
-	<date>2024-08-12T12:17:30Z</date>
+	<date>2025-02-13T00:00:00Z</date>
 	<key>pfm_macos_min</key>
-	<string>11.0</string>
+	<string>12.0</string>
```

Comment:
This should go back to 11.0 because the manifest includes keys for 1.x versions of Nudge which was not limited to 12.0 and above.